admin-tools/src/webapp/vm/memory/chef_memory.vm

```diff
@@ -11,9 +11,9 @@
 	<div class="instruction">$tlang.getFormattedMessage("memory.vm.ava.nosplit", $memory)</div>
 	<form action="#toolForm("$action")" method="post">
 		<p class="act">
-			<input type="submit" class="active" name="eventSubmit_doReset" value=" $tlang.getString("memory.vm.reset") " />
-			<input type="submit" class="active" name="eventSubmit_doEvict" value=" $tlang.getString("memory.vm.evict") " />
-			<input type="submit" name="eventSubmit_doStatus" value=" $tlang.getString("memory.vm.status") " />
+			<input type="submit" role="button" class="active" name="eventSubmit_doReset" value=" $tlang.getString("memory.vm.reset") " />
```

Comment:
I think you don't need the role defined as a button here. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Best_practices):

> If a link performs the action of a button, giving the element role="button" helps assistive technology users understand the function of the element. However, a better solution is to adjust the visual design so it matches the function and ARIA role.  Where possible, it is recommended to use native HTML buttons (<button>, <input type="button">,  <input type="submit">, <input type="reset"> and <input type="image">) rather than the button role, as native HTML buttons are supported by all user agents and assistive technology and provide keyboard and focus requirements by default, without need for additional customization.

Reply:
False positive: it is a submit button input, which doesn't require a label.